packages/web/src/scss/components/Accordion/README.md

```diff
@@ -0,0 +1,47 @@
+# Accordion
+
+Accordion is a different wrapper for multiple **Collapse** items, and can only open one item.
+
+## Usage
+
+```html
+<section id="accordionExample" class="Accordion" data-toggle="accordion">
+  <article id="accordionExample_article_0" class="Accordion__item">
+    <h3 id="accordionExample_article_0_header" class="Accordion__itemHeader">
+      <button
+        type="button"
+        class="Accordion__itemToggle"
+        data-toggle="collapse"
+        data-target="accordionExample_article_0_collapse"
+        aria-expanded="false"
+        aria-controls="accordionExample_article_0_collapse"
+      >
+        toggle
+        <span class="Accordion__itemToggleSlot">
```

Comment:
this is wrong. We need to keep slots outside of the button. Please place them next to it.

Reply:
It will be fixed in the JS branch because it is a problem to maintain two versions of these files since `readme.md` and `index.html` are the same for both CSS and JS brances.